src/LTO/Transactions/Anchor.py

```diff
@@ -53,7 +53,7 @@ def toJson(self):
             "type": self.TYPE,
             "version": self.version,
             "sender": self.sender,
-            #"senderKeyType": "ed25519",
+            "senderKeyType": "ed25519",
```

Comment:
Shouldn't this be something like `self.senderKeyType || "ed25519"`? (this goes for all txs)